packages/genesys-spark-components/src/components/stable/gux-copy-to-clipboard/gux-copy-to-clipboard.scss

```diff
@@ -5,7 +5,7 @@
     );
   }
 
-  gux-icon[icon-name='copy'] {
+  gux-icon[icon-name='fa/copy-regular'] {
```

Comment:
There seems to be some duplicate icons, for example `copy` and `fa-copy-regular` are the same visually.

Reply:
Are we switching to Font Awesome icons then? I thought we were prioritizing ours. Might be something to ask UX

Reply:
Confirmed in standup to use `fa`